### PR TITLE
test: e2e - fix/tag failing tests (maintenance)

### PR DESCRIPTION
**Pull request by ljagiela**

### CODE REVIEW

**Review by mirceahasegan — APPROVED** [review]
looks good to me.

File: packages/e2e-tests/src/assert/settings/MessageSigningAllDoneDrawerAssert.ts [hunk 1]
```diff
@@ -14,10 +14,7 @@ class MessageSigningAllDoneDrawerAssert {
     await MessageSigningAllDoneDrawer.image.waitForDisplayed();
     await MessageSigningAllDoneDrawer.title.waitForDisplayed();
     expect(await MessageSigningAllDoneDrawer.title.getText()).to.equal(await t('core.signMessage.successTitle'));
-    await MessageSigningAllDoneDrawer.description.waitForDisplayed();
-    expect(await MessageSigningAllDoneDrawer.description.getText()).to.equal(
```
Comment (DominikGuzei) [inline packages/e2e-tests/src/assert/settings/MessageSigningAllDoneDrawerAssert.ts:18]: because description was removed from UI?

**Comment by pczeglik-iohk** [comment]
# Allure Report
[`allure-report-publisher`](https://github.com/andrcuns/allure-report-publisher) generated test report!



**processReports**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/smoke/linux/chrome/7650/12276508741/index.html) for [e6dc85ee](https://github.com/input-output-hk/lace/pull/1587/commits/e6dc85ee678a50f2a91f7fcfcb4b5c18c518e046)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 33     | 0      | 4       | 0     | 37    | ✅     |

File: packages/e2e-tests/src/assert/settings/MessageSigningAllDoneDrawerAssert.ts (hunk 1 above)
Comment (ljagiela) [reply packages/e2e-tests/src/assert/settings/MessageSigningAllDoneDrawerAssert.ts:18]: correct, it's expected

**Pull request merged**